### PR TITLE
Add dragonflye task to theiaprok_ont workflow

### PR DESCRIPTION
**Pull request by cimendes**
This PR adds the dragonflye task and integrates it into the  theiaprok_ont workflow

### CODE REVIEW

File: workflows/theiaprok/wf_theiaprok_ont.wdl [hunk 1]
```diff
@@ -2,7 +2,7 @@ version 1.0
 
 import "../utilities/wf_read_QC_trim_ont.wdl" as read_qc
 import "../utilities/wf_merlin_magic.wdl" as merlin_magic_workflow
-#import "../../tasks/assembly/task_dragonflye.wdl" as dragonflye_task
+import "../../tasks/assembly/task_dragonflye.wdl" as dragonflye_task
```
Comment (frankambrosio3) [inline workflows/theiaprok/wf_theiaprok_ont.wdl:5]: I like the notation of adding "_task" to the end of tasks to differentiate them from sub wf's. Might be worth including in a round of reformatting at some point.

Comment (frankambrosio3) [reply workflows/theiaprok/wf_theiaprok_ont.wdl:5]: it also helps differentiate the task file name from the particular task within the task file.

**Review by frankambrosio3 — APPROVED** [review]
🥇

File: workflows/theiaprok/wf_theiaprok_ont.wdl (hunk 1 above)
Comment (sage-wright) [reply workflows/theiaprok/wf_theiaprok_ont.wdl:5]: I can go through and reformat when I merge 👍

**Review by sage-wright — APPROVED** [review]
awesome job! 🥇

**Pull request merged**